doculab/docs/xero.textile

```diff
@@ -132,6 +157,6 @@ h3. What happens if a Chargify-created Tax Rate is deleted in Xero?
 
 The short answer is: nothing too bad.  We detect that the tax rate has "gone away", and we re-sync it with Xero.  However, this will have the effect of creating a different Tax Rate in Xero, with a different name, but with the same rates.  Since this will segment your tax reports unnecessarily, we recommend that you do not delete the rates we create in Xero.
 
-h3. How are different currencies handled?
+h3. How many Chargify Sites can I connect to my Xero Organization
 
-At this time, we do not specify a currency when we sync billing activity.  This means that Xero will assume the transactions and payments are in the default currency for your Xero Organization.  You will need to make sure that the Invoice Account and Payment Account you select are accounts that use your default currency, which in turn must match the currency selected for your Chargify Site.
\ No newline at end of file
+There is a limit of just one Chargify Site per Xero Organization.  If you attempt to connect a second Chargify Site to the same organzation, the first Site will lose its sync privileges.  This is a limitation in the Xero authentication mechanism.  We may be able to improve upon this in the future and allow multiple Chargify Sites to use the same authentication credentials.0
```

Comment:
I think line 160 needs a question mark at the end.

Minor style things:  I would spell out 'one' on line 24, and put a comma after 2015 on line 58.